README.md

```diff
@@ -87,31 +88,36 @@ OR
+### Setting up Google OAuth Client-Id for Fence
 
- If you do not already have Google API Credentials, follow the steps below to set them up. See image below for an example on a sample Google account.
+This Docker Compose setup requires Google API Credentials in order for Fence microservice to complete its authentication. 
+To set up Google API Credentials, go to [the Credentials page of the Google Developer Console](https://console.developers.google.com/apis/credentials) and click the 'Create Credentials' button. Follow the prompts to create a new OAuth Client ID for a Web Application. Add  `https://localhost/user/login/google/login/` OR `https://YOUR_REMOTE_MACHINE_DOMAIN/user/login/google/login/` to your Authorized redirect URIs in the Credentials and click 'Create'. Then copy your client ID and client secret and use them to fill in the 'google.client_secret' and 'google.client_id' fields in the `Secrets/fence-config.yaml` file. 
+See image below for an example on a sample Google account.
 
-![Redirection Set up](Authorized%20URL.jpg)
+![Redirection Set up](Authorization_URL_2020.jpg)
 
-### Setting up Google OAuth Client-Id for Fence
-
-To set up Google API Credentials, go to [the Credentials page of the Google Developer Console](https://console.developers.google.com/apis/credentials) and click the 'Create Credentials' button. Follow the prompts to create a new OAuth Client ID for a Web Application. Add  `https://localhost/user/login/google/login/` OR `https://YOUR_REMOTE_MACHINE_DOMAIN/user/login/google/login/` to your Authorized redirect URIs in the Credentials. Then copy your client ID and client secret and use them to fill in the 'google.client_secret' and 'google.client_id' fields in the `Secrets/fence-config.yaml` file.
+If you have Google API credentials set up already that you would like to use with the local gen3 Docker Compose setup, simply add `https://localhost/user/login/google/login/` OR `https://YOUR_REMOTE_MACHINE_DOMAIN/user/login/google/login/` to your Authorized redirect URIs in your credentials and copy your client ID and client secret from your credentials to the 'client_secret' and 'client_id' fields in the `Secrets/fence-config.yaml` under `OPENID_CONNECT` and `google`.
 
 ### Setting up Users
-To set up user privileges for the services, please edit the `Secrets/user.yaml` file, following the example format shown in the file. In particular, you should change `yourlogin@gmail.com` to the email you intend to log in with, so that you can create administrative nodes later on.
+To set up user privileges for the services, please edit the `Secrets/user.yaml` file, following the example format shown in [this file](https://github.com/uc-cdis/fence/blob/master/docs/base_user.yaml). In particular, you should change all occurrences of `yourlogin@gmail.com` to the email you intend to log in with, so that you can create administrative nodes later on.
```

Comment:
i would like to the guide https://github.com/uc-cdis/fence/blob/master/docs/user.yaml_guide.md instead of the base user.yaml - or both

Reply:
oke used the one suggested!